### PR TITLE
[feat] add codeium extension

### PR DESCRIPTION
**Pull request by appelgriebsch**
re-activate `codeium`extension (free alternative to GH Copilot), which is maintained by Exafunction now

### CODE REVIEW

File: lua/lazyvim/plugins/extras/coding/codeium.lua [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  {
+    "nvim-cmp",
+    dependencies = {
+      {
+        "Exafunction/codeium.nvim",
+        opts = {},
+        config = function(_, opts)
+          require("codeium").setup(opts)
+        end,
+      },
```
Comment (folke) [inline lua/lazyvim/plugins/extras/coding/codeium.lua:20]: This can probably be removed?

**Review by folke — CHANGES_REQUESTED** [review]

File: lua/lazyvim/plugins/extras/coding/codeium.lua (hunk 1 above)
Comment (appelgriebsch) [reply lua/lazyvim/plugins/extras/coding/codeium.lua]: japp, done.

Comment (folke) [reply lua/lazyvim/plugins/extras/coding/codeium.lua]: but you still need `opts = {}` right? :) 
Otherwise setup would not be called

Comment (appelgriebsch) [reply lua/lazyvim/plugins/extras/coding/codeium.lua]: But `opt` is there on line 8 already?! Tested it locally with success.

Comment (folke) [reply lua/lazyvim/plugins/extras/coding/codeium.lua]: Nevermind, I only saw the diff from the github notif. All good! :)

**Comment by folke** [comment]
Thanks!

**Pull request merged**